images/nginx/rootfs/build.sh

```diff
@@ -35,15 +35,6 @@ export NGINX_DIGEST_AUTH=1.0.0
 # Check for recent changes: https://github.com/yaoweibin/ngx_http_substitutions_filter_module/compare/v0.6.4...master
 export NGINX_SUBSTITUTIONS=b8a71eacc7f986ba091282ab8b1bbbc6ae1807e0
 
-# Check for recent changes: https://github.com/opentracing-contrib/nginx-opentracing/compare/v0.19.0...master
```

Comment:
Please keep these in here, though we use the nginx-1.25 build for 1.10 builds we still support opentracing for 1.9 and 1.8 builds